src/Dynamo.All.sln

```diff
@@ -237,6 +237,8 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NodeAutoCompleteViewExtensi
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Md2Html", "Tools\Md2Html\Md2Html.csproj", "{0893F745-CB1A-427A-8E87-CA927273039A}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TestLinterExtension", "TestLinterExtension\TestLinterExtension.csproj", "{74A99E4A-671E-4B99-B8E9-602AE5A86323}"
```

Comment:
I think the project TestLinterExtension can live in Tests folder next to ViewExtensionLibraryTests and to preserve the naming convention there let's name it LintterExtensionTests or similar.

Reply:
if this project is going to stick around we'll need to update the test jobs to run it, we should also add it to the mono.sln.

Reply:
The idea was to pull this out when moving out of draft. I really only included it so it was easy to do some manual testing. I was thinking that it would serve as a good starting point for the sample extension that would go into `DynamoSamples`

Reply:
Other than samples I would like to keep this around to serve as a minimal set of tests for LinterExtension, but we can deal with this a little bit later, add it to mono and maybe add more tests.

Reply:
Ok, let's do like you said, the tests under LinterManager should suffice for now I think.